File: spoj/CANDY3.py
T = int(raw_input())
while T>0:
	N = int(raw_input())
	K = N
	sumi=0
	while N>0:
		x = int(raw_input())
		sumi+=x
		N-=1
	if sumi%K==0:
		print "YES"
	else:
		print "NO"
	T-=1

File: spoj/CANDY3.cpp
#include<bits/stdc++.h>
#define ll long long int
using namespace std;
int main(){
	ll Test_case,sums;
	cin>>Test_case;
	while(Test_case--){
		ll Nums,co;
		sums=0;
		cin>>Nums;
		for(ll itr=1;itr<=Nums;itr++){
			cin>>co;
			sums+=co%Nums;
		}
		if(sums%Nums==0)cout<<"YES"<<endl;
		else cout<<"NO"<<endl;
	}
	return 0;
}
